janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseCompatLoader.java

```diff
@@ -52,8 +53,8 @@ public synchronized static HBaseCompat getCompat(String classOverride) {
             String hbaseVersion = VersionInfo.getVersion();
             for (String supportedVersion : HBASE_SUPPORTED_VERSIONS) {
                 if (hbaseVersion.startsWith(supportedVersion + ".")) {
-                    if (hbaseVersion.startsWith(HBASE_VERSION_1_STRING)) {
-                        // All HBase 1.x maps to HBaseCompat1_0 for now.
+                    if (hbaseVersion.startsWith(HBASE_VERSION_1_STRING) || hbaseVersion.startsWith(HBASE_VERSION_2_STRING)) {
```

Comment:
Since both versions match with HBaseCompat1_0, should we just remove the if check?

Reply:
@hsaputra  Thanks for the review.  I will clean it up.